log4j-core/src/main/java/org/apache/logging/log4j/core/lookup/MapLookup.java

```diff
@@ -122,14 +122,14 @@ public String lookup(final LogEvent event, final String key) {
         if (map == null && !isMapMessage) {
             return null;
         }
```

Comment:
I don't think this block saves us anything because we do the same checks below -- I'd remove it.